src/main/main.ts

```diff
@@ -27,9 +27,12 @@ import { getInstalledActivities } from '../utils/getInstalledActivities'
 import { runCommand } from '../utils/runCommand'
 
 // Auto update
-require('update-electron-app')({
-  repo: 'will-stone/browserosaurus',
-})
+// disables if no apple developer id for signing app
+if (process.env.APPLE_ID) {
```

Comment:
I'm not sure this will work without changing the `make` script to
```
"make": "export $(cat .env | xargs) && electron-forge make --skip-package",
```

Reply:
In dev environment it is enough to use `package` to test package. I think it is not necessary to use `make` except publishing new version. What do you think?

Reply:
Nah, when I make I need that to be in place.

Reply:
Actually, yeah sorry, you are right. I always get those commands around the wrong way. This is cool.